cypress/e2e/home.cy.js

```diff
@@ -0,0 +1,10 @@
+describe('Home page', () => {
```

Comment:
Thanks for this, I can see exactly how to mold other tests by looking at this one.